### PR TITLE
Dxoc 507 make environment required

### PR DESCRIPTION
**Pull request by justin-fiedler**
# Amplitude Developer Docs PR

https://amplitude.atlassian.net/browse/DXOC-507

## Description

* Updated Load() docs to call out `environment` as Required.
* Updated all Ampli overviews to the same text
* Fixed swift code snippet language
* Table whitespace clean up

image

## Change type

- [x] Doc bug fix
- [x] Doc update.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

**Comment by justin-fiedler** [comment]
Linter warnings are for links in an include. Links work locally.

**Review by ZahraTee — APPROVED** [review]

File: docs/data/sdks/typescript-browser/ampli.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 <!--vale off-->
 !!!info "Ampli Browser Resources"
-    [:material-language-typescript: Ampli TypeScript Example](https://github.com/amplitude/ampli-examples/tree/main/browser/typescript/v2/react-app) · [:material-language-javascript: Ampli JavaScript Example](https://github.com/amplitude/ampli-examples/tree/main/browser/javascript/v2/react-app) · [:material-code-tags-check: Releases](https://www.npmjs.com/package/@amplitude/ampli?activeTab=versions)
+    [:material-language-typescript: TypeScript Example](https://github.com/amplitude/ampli-examples/tree/main/browser/typescript/v2/react-app) · [:material-language-javascript: JavaScript Example](https://github.com/amplitude/ampli-examples/tree/main/browser/javascript/v2/react-app) · [:material-code-tags-check: Releases](https://www.npmjs.com/package/@amplitude/ampli?activeTab=versions)
 <!--vale on-->
 
+--8<-- "includes/ampli-vs-amplitude-link-to-core-sdk.md"
+    Click here for more documentation on the [Amplitude Browser SDK](./index.md).
```
Comment (caseyamp) [inline docs/data/sdks/typescript-browser/ampli.md:16]:
```suggestion
    Visit the [Amplitude Browser SDK](./index.md) documentation.
```

**Review by caseyamp — APPROVED** [review]
@justin-fiedler this is great. I mostly made style alignments and eliminated passive voice and future tense where I could. I had a question about `environment` being required unless client.instance was included, but it's not critical. If you have any nits about my edits please let me know. 

As long as your peers believe this is technically sound, feel free to merge when you want!

**Comment by justin-fiedler** [comment]
Thanks @caseyamp ! Appreciate the suggestions, they look great. 

The "environment. Required, unless client.instance is provided" is kinda tricky, would love suggestions on how to word it.

I view `client.apiKey` and `client.instance` as advanced configuration and want promote usage of environment, hence "Required.... unless"

**Comment by justin-fiedler** [comment]
@caseyamp web preview failed for some reason. Can I merge anyway?

**Pull request merged**